content/visualize/visualize_2.md

```diff
@@ -0,0 +1,30 @@
++++
+title = "Preset"
+weight = 2
```

Comment:
Can we change the text on the Visualization intro tab too, pls? Currently it only refers to Tableau.